tests/test_ec2/test_network_acls.py

```diff
@@ -275,3 +275,32 @@ def test_duplicate_network_acl_entry():
             rule_number
         )
     )
+
+
+@mock_ec2
+def test_describe_network_acls():
+    conn = boto3.client("ec2", region_name="us-west-2")
+
+    vpc = conn.create_vpc(CidrBlock="10.0.0.0/16")
+    vpc_id = vpc["Vpc"]["VpcId"]
+
+    network_acl = conn.create_network_acl(VpcId=vpc_id)
+
+    network_acl_id = network_acl["NetworkAcl"]["NetworkAclId"]
+
+    resp = conn.describe_network_acls(NetworkAclIds=[network_acl_id])
+    result = resp["NetworkAcls"]
+
+    result.should.have.length_of(1)
+    result[0]["NetworkAclId"].should.equal(network_acl_id)
+
+    resp2 = conn.describe_network_acls()["NetworkAcls"]
+    resp2.should.have.length_of(len(resp2))
```

Comment:
This should probably be `should.have.length_of(1)` :)

Reply:
@bblommers Thank you for your reply and attention.

Reply:
Well, don't thank me yet - looks like the tests are failing now because of this change!
https://travis-ci.org/github/spulec/moto/builds/688611941

Reply:
Hi @bblommers. I found out what is happening.

When you execute `create_network_acl` method, are created 2 Network ACLs. The first is marked as a default and other is not.
When AWS account is created, consequently the default Network ACL is also created.
Therefore, in this test scenario the result should be 3.
This can also be seen in the test the `create_network_acl` method where the result is 3, [follow the code](https://github.com/spulec/moto/blob/80b64f9b3ff57515db1fc07329bf8e5f519597aa/tests/test_ec2/test_network_acls.py#L20).